src/modules/household/components/EditHouseholdMemberTable.tsx

```diff
@@ -199,18 +223,54 @@ const EditHouseholdMemberTable = ({
       },
       HOUSEHOLD_MEMBER_COLUMNS.assignedUnit(currentMembers),
       {
-        header: '',
-        key: 'action',
-        width: '1%',
+        ...BASE_ACTION_COLUMN_DEF,
         render: (hc: HouseholdClientFieldsFragment) => (
-          // No extra perm check is required, because this button only allows removing WIP Enrollments,
-          // which only requires Can Edit Enrollments, which is already required for this page
-          <RemoveFromHouseholdButton
-            currentDashboardEnrollmentId={currentDashboardEnrollmentId}
-            householdClient={hc}
-            onSuccess={refetchHousehold}
-            householdSize={currentMembers.length}
-            ariaLabel={`Remove ${clientBriefName(hc.client)}`}
+          <TableRowActions
+            record={hc}
+            recordName={clientBriefName(hc.client)}
+            MenuProps={{
+              MenuListProps: {
+                dense: false,
```

Comment:
I'm not seeing a visual difference when toggling this, what is it doing?

Reply:
Dense (default):
![Screenshot 2025-02-11 at 9 56 05 AM](https://github.com/user-attachments/assets/9f99e09a-7e32-4583-af3d-53a546e5272a)

Not dense:
![Screenshot 2025-02-11 at 9 55 36 AM](https://github.com/user-attachments/assets/18dd9956-af6b-4429-9dba-f954cd049df8)